lldb/test/API/functionalities/disassembler-variables/TestVariableAnnotationsDisassembler.py

```diff
@@ -105,6 +117,8 @@ def test_loop_reg_rotate(self):
         self.assertNotIn("<decoding error>", out)
 
     @no_debug_info_test
+    @skipIfWindows
+    @skipUnlessPlatform(["linux", "freebsd", "netbsd"])
```

Comment:
We could just skipIfWindows but then there are non-Windows COFF platforms like AIX.

Tricky to get an ideal skip here because we don't have a skip if ELF. Certainly you can remove the skipIfWindows, it's implied by the skipUnlessPlatform.

You could add `"android"` there as well. Not sure it will actually work there but I see some other tests with:
```
@skipUnlessPlatform(["linux", "android", "freebsd", "netbsd"])
```
And someone from that project can remove it later if it does fail there.